components/Contact.vue

```diff
@@ -1,28 +1,37 @@
 <template>
-  <div class="fixed left-0 w-screen h-full flex items-center justify-center">
-    <img class="absolute bottom-0 right-0" src="../static/contact/blackblob.png">
-    <img class="absolute bottom-0 left-0 select-none" src="../static/contact/greenblob.png">
-    <div class="mx-24 w-11/12 z-20">
-      <div class="font-bold text-6xl mb-10">Get in touch</div>
-      <div class="rounded-3xl bg-app-gray-2 flex py-8">
-        <div class="relative rounded-3xl w-1/3">
-          <img class="absolute left-0 bottom-0 object-cover transform origin-bottom-left scale-125" src="../static/contact/bunchofleafs.png">
-          <img class="absolute bottom-0 object-cover" src="../static/contact/unboggled.png">
+  <div class="w-full flex justify-center">
+    <div class="relative mx-auto my-16 w-11/12 z-20">
+      <div class="relative font-bold z-30 text-6xl mb-10 text-center lg:text-left">Get in touch</div>
+      <div class="absolute top-0 w-full flex justify-center">
+      <img draggable=false class="transform -translate-y-24 sm:-translate-y-40 translate-x-0 lg:translate-x-48" src="../static/contact/leaves.png">
+      </div>
+      <div class="relative rounded-3xl bg-app-gray-2 flex flex-col-reverse lg:flex-row justify-center lg:justify-end items-center">
+        <div class="relative lg:absolute lg:left-0 w-full h-full rounded-3xl flex items-end justify-center lg:justify-start select-none">
+          <img 
+            draggable=false 
+            class="hidden lg:block lg:absolute lg:left-0 object-cover z-10 w-1/3 h-full rounded-2xl" 
+            src="../static/contact/bunchofleaves.png"
+          >
+          <img 
+            draggable=false 
+            class="lg:pl-6 lg:pt-6 object-contain z-20 w-1/2 lg:w-1/4 max-h-full" 
+            src="../static/contact/unboggled.png"
```

Comment:
you can just do `./contact/unboggled.png` when referencing assets in static

Reply:
Using the path `./contact/unboggled.png` doesn't work and the image isn't found. (I tried `./static/contact/unboggled.png` as well). 
![image](https://user-images.githubusercontent.com/37466516/96781208-d5a5b000-13a1-11eb-999c-e0fa61b7ec5e.png)

Reply:
Yeah sorry no `.` should be just `/contact/unboggled.png`